targets/ramips-mt7620

```diff
@@ -32,6 +32,11 @@ device('nexx-wt3020-8m', 'wt3020-8M', {
 })
 
 
+-- TP-Link
+
+device('tp-link-archer-c2-v1', 'tplink_c2-v1')
```

Comment:
There's no installation process which utilizes the factory image. So we should not include it here.

Reply:
Good Point. No. I even tried the Snapshot OpenWRT Factory Image after reverting to stock and it wont work.